### PR TITLE
CIRC-5837 - IRONdb 0.19.12 Changelog

### PR DESCRIPTION
**Pull request by pamaddox**

### CODE REVIEW

File: content/irondb/release-notes.md [hunk 1]
```diff
@@ -5,6 +5,23 @@ weight: 40
...
 
+## Changes in 0.19.12
+
+2020-10-22
+
+ * Fix race condition that led to a potential
+   use after free.
+ * Fix bug when populating check tag ART maps where we
+   could occasionally set errornous account ids in the map,
+   causing `find` queries to miss valid results.
```
Comment (esproul) [inline content/irondb/release-notes.md]:
```suggestion
 * Fix bug when populating check tag indexes that could cause `find` queries
    to miss valid results.
```

File: content/irondb/release-notes.md [hunk 2]
```diff
@@ -5,6 +5,24 @@ weight: 40
...
 # Release Notes
 
+## Changes in 0.19.12
+
+2020-10-22
+
+ * Fix race condition that led to a potential
+   use after free.
+ * Fix bug when populating check tag ART maps where we
+   could occasionally set errornous account ids in the map,
```
Comment (bdunavant) [inline content/irondb/release-notes.md:15]:
```suggestion
   could occasionally set erroneous account ids in the map,
```

File: content/irondb/release-notes.md [hunk 3]
```diff
@@ -5,6 +5,23 @@ weight: 40
...
+   causing `find` queries to miss valid results.
+ * Fix artmap compaction bug where we were potentially losing
+   entries, leading to incomplete find results.
```
Comment (esproul) [inline content/irondb/release-notes.md:18]:
```suggestion
 * Fix tag search index compaction bug that could cause incomplete find
    results.
```

**Review by bdunavant — CHANGES_REQUESTED** [review]

File: content/irondb/release-notes.md [hunk 4]
```diff
@@ -5,6 +5,23 @@ weight: 40
...
+ * Fix error in tag index iteration that could cause incomplete
+   find results.
```
Comment (esproul) [inline content/irondb/release-notes.md:20]:
```suggestion
 * Fix error in tag search index iteration that could cause incomplete find
    results.
```

File: content/irondb/release-notes.md [hunk 5]
```diff
@@ -5,6 +5,23 @@ weight: 40
...
+ * Add `explain=1` option to `/find//tags` endpoint. Returns a header
+   explaining the full query that was performed on each node.
+ * Added support for cumulative histogram ingestion.
```
Comment (esproul) [inline content/irondb/release-notes.md:23]:
```suggestion
 * Add support for cumulative histogram ingestion.
```

File: content/irondb/release-notes.md [hunk 6]
```diff
@@ -5,6 +5,24 @@ weight: 40
...
 
+## Changes in 0.19.12
+
+2020-10-22
+
+ * Fix race condition that led to a potential
+   use after free.
+ * Fix bug when populating check tag ART maps where we
+   could occasionally set erroneous account ids in the map,
+   causing `find` queries to miss valid results.
```
Comment (esproul) [inline content/irondb/release-notes.md:16]:
```suggestion
 * Fix bug when populating check tag search indexes that could cause incomplete
    find results.
```

**Review by esproul — CHANGES_REQUESTED** [review]
Suggestions for more user-friendly wording.

File: content/irondb/release-notes.md [hunk 7]
```diff
@@ -5,6 +5,24 @@ weight: 40
...
+
+ * Fix race condition that led to a potential use after free.
+ * Fix bug when populating check tag search indexes that could cause incomplete
+   find results.
+ * Fix tag search index compaction bug that could cause incomplete find
+   results.
+ * Fix error in tag search index iteration that could cause incomplete find
+   results.
+ * Fix bug when populating check tag indexes that could cause `find` queries
+   to miss valid results.
```
Comment (esproul) [inline content/irondb/release-notes.md:20]: Looks like a dupe?

**Review by esproul — CHANGES_REQUESTED** [review]
Possible dupe

File: content/irondb/release-notes.md (hunk 7 above)
Comment (pamaddox) [reply content/irondb/release-notes.md:20]: Not actually a dupe in theory - they were separate bugs - one bug in population, a separate bug in iteration. To users, however, it's likely all the same, so I'll consolidate.

**Review by esproul — APPROVED** [review]
LGTM

**Pull request merged**